### PR TITLE
Removed Deprecated VF2 Call Limit

### PR DESCRIPTION
**Pull request by sbrandhsn**
### Summary

Removed deprecated `get_vf2_call_limit` function and associated tests.

### CODE REVIEW

File: releasenotes/notes/deprecation-passmanager-0.25-95eb9b45b517370a.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+---
+upgrade:
+  - |
+    Removed deprecated function :func:`~qiskit.transpiler.preset_passmanagers.common.get_vf2_limits` and its corresponding test.
```
Comment (kevinhartman) [inline releasenotes/notes/deprecation-passmanager-0.25-95eb9b45b517370a.yaml]:
```suggestion
    Removed deprecated function :func:`~qiskit.transpiler.preset_passmanagers.common.get_vf2_call_limit` and its corresponding test.
```

Typo in the function name, that's the replacement 😄.

Comment (kevinhartman) [reply releasenotes/notes/deprecation-passmanager-0.25-95eb9b45b517370a.yaml]: We tend to also include one last reminder of what should be used instead for a removed deprecation in the upgrade notes. For example, adding something like:

```
Instead, use :func:`~qiskit.transpiler.preset_passmanagers.common.get_vf2_limits`
```

 to the end of what you've already got.

Comment (sbrandhsn) [reply releasenotes/notes/deprecation-passmanager-0.25-95eb9b45b517370a.yaml]: Thanks for spotting these and for your suggestion! I updated the release note. :-)

File: releasenotes/notes/deprecation-passmanager-0.25-95eb9b45b517370a.yaml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+---
+upgrade:
+  - |
+    Removed deprecated function :func:`~qiskit.transpiler.preset_passmanagers.common.get_vf2_call_limit`
+    and its corresponding test. Instead, use
```
Comment (kevinhartman) [inline releasenotes/notes/deprecation-passmanager-0.25-95eb9b45b517370a.yaml:5]: One more small point for release note style is that we ought to only mention user-facing changes, so in this case we can omit the bit about the tests being updated. Not a reason to hold up this PR, we can easily adjust this when we create and review the release notes PR during release 🙂.

**Review by kevinhartman — APPROVED** [review]
Looks good, thanks for doing this removal!

File: releasenotes/notes/deprecation-passmanager-0.25-95eb9b45b517370a.yaml (hunk 2 above)
Comment (sbrandhsn) [reply releasenotes/notes/deprecation-passmanager-0.25-95eb9b45b517370a.yaml:5]: Sounds good, I will keep this in mind the next time I write a release note, thanks! :-)

**Pull request merged**